package.json

```diff
@@ -12,9 +12,12 @@
     "ui:build:watch": "npm --prefix ui run build:watch",
     "prettier": "prettier .circleci/config.yml  functions/{**/,}*.{ts,json} ui/{**/,}*.{ts,json,html}",
     "format:check": "npm run prettier -- --list-different",
-    "format:fix": "npm run prettier -- --write"
+    "format:fix": "npm run prettier -- --write",
+    "test": "ava ./deployment/check-config/**.test.js",
+    "config:check": "node ./deployment/check-config/check-config.js ./config.json "
```

Comment:
I wouldn't put the config.json argument here since it is the CI that decides it is named that way.